packages/nuxt-locokit/playground/nuxt.config.ts

```diff
@@ -1,17 +1,29 @@
-import { defineNuxtConfig } from 'nuxt'
+import { defineNuxtConfig } from 'nuxt/config'
 import LocoKitModule from '..'
 
 export default defineNuxtConfig({
+  // build: {
+  //   transpile: ['vee-validate, @vee-validate/rules'],
+  // },
```

Comment:
Could you explain these lines ?

Reply:
It was just a test when I was lost :)

I have remove it